content/en/docs/setup/install/helm/index.md

```diff
@@ -183,6 +183,14 @@ to manage the lifecycle of Istio.
     {{< text bash >}}
     $ helm init --service-account tiller
     {{< /text >}}
+    
+    
+1. Create a namespace for the `istio-system` components:
```

Comment:
helm (should) create the namespace for you with the installation of the helm init chart.